damus/Views/Images/FullScreenCarouselView.swift

```diff
@@ -96,8 +96,23 @@ struct FullScreenCarouselView<Content: View>: View {
                 GeometryReader { geo in
                     VStack {
                         if showMenu {
-                            NavDismissBarView(navDismissBarContainer: .fullScreenCarousel)
-                                .foregroundColor(.white)
+                            
+                            HStack {
+                                NavDismissBarView(navDismissBarContainer: .fullScreenCarousel)
+                                    .foregroundColor(.white)
+                                
+                                if let url = urls[safe: selectedIndex] {
+                                    ShareLink(item: url.url) {
```

Comment:
From testing, this seems to be sharing only the URL of the image/video, which is inconsistent with the other PR, which shared the image object itself. Besides, Sharing an URL seems to reduce the number of available sharing options (e.g. I can't save it to my Photos app)

Is there a way we can share the image and/or video instead of the URL?

Reply:
The image itself wasn’t directly accessible within the same struct, so I made some adjustments in the new commit to capture the actual image content by passing a state variable. However, this approach doesn’t apply to video content due to differences in how it’s displayed. Thanks for pointing that out! @danieldaquino